tests/test_measures_inference.py

```diff
@@ -9,6 +11,44 @@
 
 BLDG = Namespace("https://brickschema.org/schema/ExampleBuilding#")
 
+
+def _get_subclasses(hierarchy):
+    """
+    Helper method to generate a list of pairwise
+    subclass relationships denoted by a hierarchy definition in 'bricksrc'
+    """
+    for klass, defn in hierarchy.items():
+        if 'subclasses' not in defn:
+            continue
+        for subklass, subdefn in defn['subclasses'].items():
+            yield (klass, subklass)
+        for tup in _get_subclasses(defn['subclasses']):
+            yield tup
+
+
+def test_measurable_hierarchy():
```

Comment:
I am fine with this, but shouldn't this test cover any classes than just substances/quantities? (and I believe they should be handled in the same way as I modified in the https://github.com/BrickSchema/Brick/pull/98.)

Reply:
I like factoring the substances and quantity tests into their own case, but I can expand these test cases for the rest of the hierarchy as well.